### ISSUE TITLE
Look at simplifying the API as it is too complex.

### ISSUE THREAD

**Issue by RyanONeill1970** [TheGeneGenieProject/GeneGenie.Gedcom#14]
To add an individual to the in memory database we just add a new GedcomIndividualRecord.
To add sub records (birth, names etc) we add those to the individual but also have to pass the db as well.

We could adopt a fluent API approach and also have it so that when a sub record is added, it inherits the database from the parent record.

**Comment by HolyOne** [comment]
I agree. At least helper methods like  GetSibblings() , GetParents()  wıould be nice.
Also calculating the relationship between 2 individuals would be helpful (Like x is 3rd cousin of y)
